src/Input/index.tsx

```diff
@@ -67,6 +70,8 @@ const Input: FunctionComponent<Props> = forwardRef<HTMLInputElement, Props>(func
           name={name}
           disabled={disabled}
           onChange={formControl}
+          onBlur={formControl}
+          value={value}
```

Comment:
I just remembered that you'll need to add the useControlled hook to this to manage controlled vs uncontrolled